.github/workflows/inspektor-gadget.yml

```diff
@@ -1473,13 +1471,91 @@ jobs:
+    runs-on: ubuntu-latest
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint.
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [amd64, arm64]
+    env:
+      AWS_REGION: us-east-2
+      AWS_EKS_CLUSTER_PREFIX: ig-ci-eks-
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup go
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ env.GO_VERSION }}
+        cache: true
+    - name: Set container repository and determine image tag
+      id: set-repo-determine-image-tag
+      uses: ./.github/actions/set-container-repo-and-determine-image-tag
+      with:
+        registry: ${{ env.REGISTRY }}
+        container-image: ${{ env.CONTAINER_REPO }}
+    - name: Craft cluster name
+      shell: bash
+      run: |
+        echo "CLUSTER_NAME=${{ env.AWS_EKS_CLUSTER_PREFIX }}${{ matrix.arch }}-${RANDOM}" >> $GITHUB_ENV
+    # Install eksctl following https://eksctl.io/installation/#for-unix
+    - name: Install eksctl
+      run: |
+        ARCH=$(dpkg-architecture -qDEB_HOST_ARCH)
+        PLATFORM=$(uname -s)_$ARCH
+        curl -sLO "https://github.com/eksctl-io/eksctl/releases/latest/download/eksctl_$PLATFORM.tar.gz"
+        curl -sL "https://github.com/eksctl-io/eksctl/releases/latest/download/eksctl_checksums.txt" | grep $PLATFORM | sha256sum --check
+        tar -xzf eksctl_$PLATFORM.tar.gz -C /tmp && rm eksctl_$PLATFORM.tar.gz
+        sudo mv /tmp/eksctl /usr/local/bin
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        role-to-assume: ${{ vars.AWS_ROLE }}
+        aws-region: ${{ env.AWS_REGION }}
+    - name: Create EKS cluster ${{ env.CLUSTER_NAME }}
+      shell: bash
+      run: |
+        # We need to use big nodes, otherwise integration tests fail because the node can't handle
+        # many gadgets running on parallel
+        node_type='t2.xlarge'
+        if [ ${{ matrix.arch }} = 'arm64' ]; then
+          node_type='a1.xlarge'
+        fi
+        eksctl create cluster --name ${{ env.CLUSTER_NAME }} --tags ig-ci=true --node-type $node_type
+    - name: Run integration tests
+      uses: ./.github/actions/run-integration-tests
+      with:
+        kubernetes_distribution: "eks-AmazonLinux"
+        kubernetes_architecture: "${{ matrix.arch }}"
+        container_repo: ${{ steps.set-repo-determine-image-tag.outputs.container-repo }}
+        image_tag: ${{ steps.set-repo-determine-image-tag.outputs.image-tag }}
+        gadget_repository: ${{ steps.set-repo-determine-image-tag.outputs.gadget-repository }}
+        gadget_tag: ${{ steps.set-repo-determine-image-tag.outputs.gadget-tag }}
+        test_summary_suffix: "EKS-${{ matrix.arch }}"
+    - name: Delete EKS cluster ${{ env.CLUSTER_NAME }}
+      if: always()
+      shell: bash
+      run: |
+        eksctl delete cluster --name ${{ env.CLUSTER_NAME }} --wait=false
```

Comment:
It seems the [`--wait=false`](https://github.com/inspektor-gadget/inspektor-gadget/actions/runs/7845825443/job/21484855822#step:10:1) isn't doing anything ?